### PR TITLE
Fix messages.

### PR DESCRIPTION
**Pull request by awholegunch**
Fixes a number of simple errors in the UI messages. Adds periods, removes spaces, restructures one of the messages...

### CODE REVIEW

**Comment by elfenheart** [comment]
:+1: LGTM

**Comment by awholegunch** [comment]
I guess we should run a build, just to make sure....

**Comment by awholegunch** [comment]
http://builds.cask.co/browse/CDAP-RBT497-1

File: cdap-ui/app/features/adapters/controllers/create/toppanel-ctrl.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ angular.module(PKG.name + '.feature.adapters')
                       });
                     }.bind(this),
                     function error() {
-                      console.log('ERROR: ' + 'exporting adapter failed');
+                      console.log('ERROR: Exporting adapter ' + MyAppDAGService.metadata.name + ' failed.');
```
Comment (gokulavasan) [inline cdap-ui/app/features/adapters/controllers/create/toppanel-ctrl.js]: There is no adapter in 3.2.

**Pull request merged**